app/components/work_package_types/pattern_input.sass

```diff
@@ -1,6 +1,7 @@
 .op-pattern-input
   &--text-field
-    white-space: pre-wrap
+    white-space: nowrap
+    overflow: hidden
```

Comment:
> do not wrap, but overflow horizontically - with scroll

This looks like we'd not enable scrolling on overflow. Can you maybe indicate which edge case this PR is fixing? In the linked ticket I don't read anything about horizontal scrolling.